packages/protocol/contracts-0.8/common/EpochManager.sol

```diff
@@ -58,7 +55,7 @@ contract EpochManager is
   address public oracleAddress;
   address[] public elected;
 
-  mapping(address => ProcessedGroup) public processedGroups;
+  mapping(address => tuint256) public processedGroups;
```

Comment:
this, very likely, breaks the storage on alfajores.

Reply:
it should actually work since we are zeroing out everything after every use